app.js

```diff
@@ -0,0 +1,16 @@
+const hamburgerOpenConst = document.querySelector('.hamburgerOpen');
+const hamburgerCloseConst = document.querySelector('.hamburgerClose');
+const menuPageConst = document.querySelector('.menuPage');
+// const pageLinkConst = document.querySelectorAll('.pageLink');
+
+function open() {
+  menuPageConst.style.display = 'flex';
+  menuPageConst.style.top = '0';
+}
+
+function close() {
+  menuPageConst.style.top = '-100%';
+}
+
+hamburgerOpenConst.addEventListener('click', open);
+hamburgerCloseConst.addEventListener('click', close);
```

Comment:
> When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

- [ ] These two requriemtn were not implemented in your project, when a user clicks on a menu item, it should take the user to that particular section of the page. Also when a user clicks on a menu item on a mobile meant, it should also close the menu. I'll suggest you implement these. :smile: